generator/src/Pos/Generator/Block/Payload.hs

```diff
@@ -134,7 +134,7 @@ genTxPayload pm = do
     flip evalStateT gtd $ do
         (a,d) <- lift $ view tgpTxCountRange
         txsN <- fromIntegral <$> getRandomR (a, a + d)
-        void $ replicateM txsN genTransaction
+        void $ replicateM_ txsN genTransaction
```

Comment:
Is the `void` still needed one you use `repliacteM_` ?

Reply:
I've been wondering the same. And I doubt it. Was just following the `hlint` suggestions blindly :-)
```haskell
.\> :t void
void :: Functor f => f a -> f ()
.\> :t replicateM_
replicateM_ :: Applicative m => Int -> m a -> m ()
.\> :t void . replicateM_
void . replicateM_ :: Applicative m => Int -> m a -> ()
```
where
```haskell
void :: Functor f => f a -> f ()
void x = () <$ x
```